mpu6050/mpu6050_cdev.h

```diff
@@ -0,0 +1,191 @@
+#include <linux/fs.h>
+#include <linux/cdev.h>
+#include <asm/uaccess.h>
+
+#include "mpu6050_data.h"
+
+
+struct cdev_instance {
+	unsigned int major;
+	dev_t dev_no;
+	dev_t read_all;
+	struct cdev *cdev;
+};
+
+struct cdevs_holder {
+	struct cdev_instance cdev_line;
+	struct cdev_instance cdev_full;
+	struct mpu6050_data_holder *data;
+};
+
+static int open_cdev(struct inode *, struct file *);
+static int release_cdev(struct inode *, struct file *);
+static ssize_t read_line(struct file *, char __user *, size_t, loff_t *);
+static ssize_t read_full(struct file *, char __user *, size_t, loff_t *);
+
+struct file_operations g_fops_line = {
+	read: read_line,
+	open: open_cdev,
+	release: release_cdev,
+	owner: THIS_MODULE
+};
+
+struct file_operations g_fops_full = {
+	read: read_full,
+	open: open_cdev,
+	release: release_cdev,
+	owner: THIS_MODULE
+};
+
+static struct cdevs_holder *get_cdevs(void)
+{
+	static struct cdevs_holder g_cdevs_holder;
+	return &g_cdevs_holder;
+}
+
+static struct cdev_instance *get_cdev(dev_t dev_no)
```

Comment:
Implementing functions in header files is not recommended.
There are some exceptions like "wrappers" functions (e.g. i2c_smbus_read_word_swapped) but they should be declared as static inline.

Reply:
Will do in PR #33